chain/cosmos/cosmos_chain.go

```diff
@@ -293,153 +299,6 @@ type ValidatorWithIntPower struct {
 	PubKeyBase64 string
 }
 
-// Bootstraps the chain and starts it from genesis
-func (c *CosmosChain) StartWithGenesisFile(testName string, ctx context.Context, home string, pool *dockertest.Pool, networkID string, genesisFilePath string) error {
```

Comment:
This was never used.

Reply:
This was used during the juno halt to test the new genesis file. That has been removed and a link to it is in the trophies in the README. Something like this could be used for the test caching, e.g. export state after creating clients, connections, and channels, to have a quick restore point, then a function similar to this could do a quick startup. Would also be nice to have something like this available for testing existing genesis files in one off test cases.